### PR TITLE
Clarifications from feedback

### PR DESCRIPTION
**Pull request by kraigb**

### CODE REVIEW

File: docs/python/linting.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ MetaSocialImage: images/tutorial/social.png
 ---
 # Linting Python in VS Code
 
-Linting highlights syntactical and stylistic errors in your Python source code. By default, linting for Python is enabled in Visual Studio Code using Pylint. If PyLint is not already installed in your selected Python environment, VS Code prompts you to install the necessary packages. (You can also manually run the Python package manager to install PyLint at a command prompt for the same environment: `pip install pylint` (Windows) or `pip3 install pylint` (Mac/Linux).)
+Linting highlights syntactical and stylistic errors in your Python source code. By default, linting for Python is enabled in Visual Studio Code using Pylint. If PyLint is not already installed in your selected Python environment, VS Code prompts you to install the necessary packages. (You can also manually run the Python package manager to install PyLint at a command prompt for the same environment: `pip3 install pylint` (Mac/Linux) or `pip install pylint` (Windows; may require elevation).)
```
Comment (gregvanl) [inline docs/python/linting.md]: We should use macOS rather than Mac. I'll fix these before I publish today.

Comment (kraigb) [reply docs/python/linting.md]: OK, I'll remember that for future. You might do a quick search/replace as Mac and Mac/Linux instances may occur elsewhere.

Comment (gregvanl) [reply docs/python/linting.md]: https://github.com/Microsoft/vscode-docs/commit/15705cdcf4def30d38fd7e67808bf2ae5a27952d

Comment (kraigb) [reply docs/python/linting.md]: Looks good, thanks!

**Pull request merged**